### PR TITLE
map in skeleton

### PR DESCRIPTION
**Pull request by JaneJiangHH**
#26

### CODE REVIEW

**Review by andreaseliasson — APPROVED** [review]
Looks good to me.

File: js-src/src/app/map.component.js [hunk 1]
```diff
@@ -1,15 +1,62 @@
...
+    We'll add all the code artifacts to this one global object.
+    Most application files export one thing by adding that thing to the app namespace. Our app.component.js file exports the AppComponent.
+*/
+
 import {Component} from '@angular/core';
 
-import mapTemplate from './map.component.html'
+import mapTemplate from './map.component.html';
+
+import '../../public/css/styles.css';
```
Comment (blanche) [inline js-src/src/app/map.component.js:10]: do we need this, when it is already imported in the app component?

File: js-src/src/app/map.component.js [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+import '../main.js';
```
Comment (blanche) [inline js-src/src/app/map.component.js]: why do we import main.js here? do we need anything from that file?

File: js-src/src/app/map.component.js [hunk 3]
```diff
@@ -1,15 +1,62 @@
...
 import mapStyling from './map.component.css';
 
+import {GOOGLE_MAPS_DIRECTIVES,GOOGLE_MAPS_PROVIDERS} from 'angular2-google-maps/esm/core';
+
 let MapComponent = Component({
+    selector: 'my-map',
```
Comment (blanche) [inline js-src/src/app/map.component.js]: also, why do we need a selector which will not be used?

Comment (JaneJiangHH) [reply js-src/src/app/map.component.js]: yeah, this line of code can be deleted

File: js-src/src/app/map.component.js (hunk 1 above)
Comment (JaneJiangHH) [reply js-src/src/app/map.component.js:10]: I will check if it will be ok without this

Comment (JaneJiangHH) [reply js-src/src/app/map.component.js:10]: comment it has no influence, I will delete these unnecessary codes and upload again

File: js-src/src/app/map.component.js (hunk 2 above)
Comment (JaneJiangHH) [reply js-src/src/app/map.component.js]: Xuan just forgot to delete this import line

**Pull request merged**